site/pages/faq.html

```diff
@@ -0,0 +1,45 @@
+---
+title: FAQ
+description: FAQ on the FT's engineering competencies
+permalink: /docs/faq
+layout: o-layout-docs
+---
+
+# {{page.title}}
+
+## Is there an individual contributor route?
+
+There is no separate individual contributor route to promotion, but the aim is
+to have a single set of competencies that are flexible enough to allow for a
+variety of ways that an engineer can demonstrate seniority.
+
+A key idea here is that the more senior you are as an engineer, the more you
+should be helping those around you to achieve. There are many ways to do
+this:
+- Line managing
+- Mentoring
+- Directing the work of a team ("tech leading")
+- Reviewing and driving up the quality in others' technical output
+- Influencing technical communities and decisions across the organisation
+
+Choosing not to pursue line management will mean you need to demonstrate
+that you are having a positive impact on others' work in other ways.
+
+## There are no competencies for Principal Engineer. Why is this?
+
+The Product & Technology Executive/Technology Leadership Group made a decision
+in 2018: Principal roles and beyond must have a position open and advertised.
+The promotions board therefore plays no role in deciding who becomes a
+Principal Engineer, so there are no competencies defined at this stage. The
+recruiting manager for each Principal Engineer role defines what the criteria
+for that role will be.
+
+There is no specific cap on the number of Principals; it is governed by open roles
+and decisions about the organisation’s structure. To advance to Principal Engineer
+either a position needs to become vacant, or PTE/TLG need to approve the
+creation of a new Principal Engineer role, which will typically only happen
```

Comment:
> PTE/TLG

Afaik this is the tech director's decision if their budget permits and it's in the interest of the tram (not necessarily due to a massive expansion either). Might be some tgl consultation but I don't think PTE are involved.

Reply:
Thanks for the info (I basically made up what was there). I've reworded to correct that, but have left in some examples of when PE roles get created; don't want to leave it at 'new role must be created', as then that begs the question of 'well - when does that happen then?'